modules/repository/check.go

```diff
@@ -72,13 +72,18 @@ func GitGcRepos(ctx context.Context) error {
 			if err := repo.GetOwner(); err != nil {
 				return err
 			}
+			log.Trace("Running git gc on %v", repo)
 			if stdout, err := git.NewCommand(args...).
 				SetDescription(fmt.Sprintf("Repository Garbage Collection: %s", repo.FullName())).
 				RunInDirTimeout(
 					time.Duration(setting.Git.Timeout.GC)*time.Second,
 					repo.RepoPath()); err != nil {
 				log.Error("Repository garbage collection failed for %v. Stdout: %s\nError: %v", repo, stdout, err)
-				return fmt.Errorf("Repository garbage collection failed: Error: %v", err)
+				desc := fmt.Sprintf("Repository garbage collection failed for %s. Stdout: %s\nError: %v", repo.RepoPath(), stdout, err)
```

Comment:
This only goes to the admin's panel, does it? Otherwise it would require sanitation.

Reply:
yes